project/Dependencies.scala

```diff
@@ -0,0 +1,82 @@
+import sbt._
+import Keys._
+
+object Dependencies {
+
+  val scalaVer = "2.12.6"
```

Comment:
What do you think about factoring the version numbers into a separate Version object?

Reply:
Moved to Versions object in same file.